### PR TITLE
first version of web-practice

### PR DESCRIPTION
**Pull request by oleksiistepaniak**

### CODE REVIEW

**Review by potrap — APPROVED** [review]
looks good :)

File: src/main/webapp/WEB-INF/views/cars/addDriver.jsp [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<h1 style="text-align:center;">Hello!</h1>
+<h2 style="text-align:center;">If you want to add a driver to car, please fill the next form:</h2>
+
+<form method="post" action="${pageContext.request.contextPath}/cars/addDriver">
+    Driver id: <input type="text" name="driver_id"><br>
+    Car id: <input type="text" name="car_id"><br>
+    <button type="submit">Confirm</button>
+</form>
+</body>
+</html>
```
Comment (petro-ushchuk) [inline src/main/webapp/WEB-INF/views/cars/addDriver.jsp:16]: add some space to the end. fix in all files.

File: src/main/webapp/web.xml [hunk 2]
```diff
@@ -11,4 +11,44 @@
...
+        <servlet-name>Creating a car</servlet-name>
+        <url-pattern>/cars/create</url-pattern>
+    </servlet-mapping>
+    <servlet>
+        <servlet-name>Adding driver to car</servlet-name>
+        <servlet-class>mate.controller.AddDriverToCarController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>Adding driver to car</servlet-name>
+        <url-pattern>/cars/addDriver</url-pattern>
```
Comment (petro-ushchuk) [inline src/main/webapp/web.xml:52]: And this one as well

**Review by petro-ushchuk — CHANGES_REQUESTED** [review]

**Review by petro-ushchuk — APPROVED** [review]
Good work 🔥

**Pull request closed without merging**